packages/participants-svc/src/domain/participant_agg.ts

```diff
@@ -241,7 +247,7 @@ export class ParticipantAggregate {
                 );
                 hubParticipant.participantAccounts.push(participantReconAccount);
                 hubParticipant.changeLog.push({
-                    changeType: ParticipantChangeTypes.ADD_ACCOUNT,
+                    changeType: ParticipantChangeTypes.CHANGE_ACCOUNT,
```

Comment:
This should be ADD or CREATE, see comment on names below

Reply:
Updated with ADD.